spec/helpers/null_logger.rb

```diff
@@ -0,0 +1,9 @@
+require 'logger' unless defined? Logger
+
+class NullLogger < Logger
```

Comment:
err...sorry. So the diff in this gem is that if you run specs, it'll barf all over STDOUT. this is just to send logs in spec to the bitbucket